Source/Projections/Store/IProjectionAssociations.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Events;
+
+namespace Dolittle.SDK.Projections.Store
+{
+    /// <summary>
+    /// Defines a associations for projections.
+    /// </summary>
+    public interface IProjectionAssociations
```

Comment:
In the spirit of naming, maybe:
```suggestion
    public interface IProjectionReadModelTypeAssociations
```